Resources/Prototypes/materials.yml

```diff
@@ -35,3 +35,12 @@
   electricresistivity: 2.1e-1
   thermalconductivity: 80
   specificheat: 2000
+
+- type: material
+  id: plasteel
+  name: plasteel
+  icon: Objects/Materials/sheet_plasteel.png
+  density: 15400 # literally arbitrary values...
+  electricresistivity: 6.9e-7
```

Comment:
Can we camelCase `electricresistivity`, `thermalconductivity` and especially `specificheat`?

Reply:
out of scope but I'll bite the bullet and do it anyway.